src/structures/game_session.ts

```diff
@@ -672,6 +673,18 @@ export default class GameSession {
         this.filteredSongs = await getFilteredSongList(guildPreference);
     }
 
+    getOwner() {
+        return this.ownerQueue[0];
+    }
+
+    transferOwner() {
+        if (this.ownerQueue.length === 1) {
+            return;
+        }
+
+        this.ownerQueue.shift();
```

Comment:
What if the next person in the queue isn't in the vc anymore?

Reply:
Should transferOwner explicitly remove the userID instead of just shifting?